bench/record/record.bash

```diff
@@ -15,6 +15,10 @@ bench_start=`env_val "${env}" 'bench.start'`
 if [ -z "${bench_start}" ]; then
 	bench_start='0'
 fi
+test_name=`env_val "${env}" "bench.${workload}.test-name"`
+if [ ! -z "${test_name}" ]; then
+	workload=${workload}-${test_name}
```

Comment:
workload has a specific meaning, suggest:
1, put test_name to tag(s) in score table
2, use a spare col for test_name (alter table to add a col) in detail table

That's what detail table for.

Reply:
`test_name` is already encoded in the `tag`, so I will just remove these lines.